test/passing/format_invalid_files_with_locations.ml.opts

```diff
@@ -0,0 +1 @@
+--format-invalid-files=auto -n 3
```

Comment:
We usually use `--max-iter` instead of `-n` we should keep it that way to grep everything in one go